ladybug/wea.py

```diff
@@ -521,7 +528,7 @@ def is_leap_year(self):
         return self._is_leap_year
 
     @staticmethod
-    def hour_count(is_leap_year):
+    def hr_count(is_leap_year):
```

Comment:
In honesty, I did this because certain very long lines of code resulted from having the full name but I agree that this is a bad reason.  I will put it back and change the long lines of code instead.